typescript/utils/src/addresses.ts

```diff
@@ -372,3 +372,8 @@ export function ensure0x(hexstr: string) {
 export function strip0x(hexstr: string) {
   return hexstr.startsWith('0x') ? hexstr.slice(2) : hexstr;
 }
+
+export const fromHexString = (hexstr: string) =>
+  Buffer.from(strip0x(hexstr), 'hex');
```

Comment:
Redundant with `addressToBytes` above

Reply:
`addressToBytes` enforces a specific length which we do not want here

Reply:
@yorhodes then it probably doesn't belong in `addresses.ts` right?